### PR TITLE
Add Sonic Boom: Shattered Crystal

### PR DESCRIPTION
**Pull request by legoj15**
Adds Sonic Boom: Shattered Crystal.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/sonic-boom-shattered-crystal/game.dat [hunk 1]
```diff
@@ -0,0 +1,14 @@
+title = "Sonic Boom: Shattered Crystal"
+description = "In Sonic Boom: Shattered Crystal, you control Knuckles, Sonic, Sticks, and Tails to save Amy after she was kidnapped. The game mainly consists of platformer levels, where you warp to multiple locations in each level. The game was published by SEGA, with an accompanying game on the Wii U, 'Sonic Boom: Rise of Lyric'"
```
Comment (Leo626) [inline games/sonic-boom-shattered-crystal/game.dat]: Use the wikipedia description, with some minor tweaks.

` Sonic Boom: Shattered Crystal, also known as \"Sonic Toon: Island Adventure\" in Japan, is an action-adventure video game developed by Sanzaru Games and published by Sega. Along with its Wii U sibling Sonic Boom: Rise of Lyric, Shattered Crystal is a spin-off of Sega's Sonic the Hedgehog franchise and is a part of the Sonic Boom sub-franchise.`

**Comment by legoj15** [comment]
That screenshot was taken from my 3DS. Should I remove it?

**Comment by Leo626** [comment]
Only screenshots taken from CItra are allowed so showing the title screen would be misleading. You said that it only shows the loading ring for a second before the screen goes black. A screenshot of the loading ring would do.

File: games/sonic-boom-shattered-crystal/game.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
+title = "Sonic Boom: Shattered Crystal"
+description = "Sonic Boom: Shattered Crystal, also known as \"Sonic Toon: Island Adventure\" in Japan, is an action-adventure video game developed by Sanzaru Games and published by Sega. Along with its Wii U sibling Sonic Boom: Rise of Lyric, Shattered Crystal is a spin-off of Sega's Sonic the Hedgehog franchise and is a part of the Sonic Boom sub-franchise.
```
Comment (Leo626) [inline games/sonic-boom-shattered-crystal/game.dat]: Missing `"` at the end of the sentence.

**Comment by legoj15** [comment]
Thanks. Addressed the issues.

**Pull request merged**